Makefile

```diff
@@ -90,6 +90,18 @@ build: ## Generate a binary in ./bin
 	@echo "==> Building binary"
 	go build -ldflags "${LINKER_FLAGS}" -o ${DESTINATION}
 
+.PHONY: e2e-test-atlas
```

Comment:
this split was not part of the ticket and like I mention before now adds to the complexity of the PR
also with the changes you're making does `make e2e-test` still works?

Reply:
yes, it is working. I decided to create an atlas folder inside the e2e because we will have to run ops_manager on e2e as well and those e2e test files must be inside different folders I guess. I have created iam just because I thought will be better to keep them separated. I will revert that change if you wish.

Reply:
the main reason of my comment is that we are adding additional changes that make the PR a bit harder to review, not only now we need to review evergreen works but like my question signals, does this break expected behaviour? no need to revert this but keep that in mind for future PRs, if there are things to improve is always a good idea to just open a new PR for it